packages/wp-babel-makepot/index.js

```diff
@@ -20,7 +20,7 @@ const mergeOptions = ( preset, options = {} ) => {
 
 module.exports = ( filepath, options = {} ) => {
 	try {
-		return babel.transformFileSync( filepath, mergeOptions( defaultPreset, options ) );
+		return babel.transformFileAsync( filepath, mergeOptions( defaultPreset, options ) );
```

Comment:
Babel doesn't support ESM when called synchronously
(I got `"You appear to be using a native ECMAScript module plugin, which is only supported when running Babel asynchronously"`).

I converted this and `cli.js` to async to support it.